vue/package.json

```diff
@@ -22,29 +22,32 @@
     "@heroicons/vue": "^2.0.13",
```

Comment:
This can also be removed, I think, if we are completed converted over to material design icons.